lib/eupspkg.sh

```diff
@@ -737,7 +737,7 @@ default_config()
 			fix_autoconf_timestamps
 		fi
 
-		./configure $CONFIGURE_OPTIONS
+	        eval ./configure $CONFIGURE_OPTIONS
```

Comment:
This is to fix the problem of quoting in `$CONFIGURE_OPTIONS`. Look at the `eupspkg.cfg.sh` file in `cfitsio`. I couldn't work out any way to sanely quote `$PREFIX` such that the `./configure` call here worked properly. The only solution was to use quotes as you'd expect in the `eupspkg.cfg.sh` file and then use `eval` here. Probably should have added a comment about that in here.